content/index.mdx

```diff
@@ -29,7 +29,7 @@ Our instructors hail from every continent but Antarctica. (That’s next.) egghe
 
 egghead was created by a small team of web developers who know how essential it is to keep learning. There’s always a new tool or framework to figure out. We made egghead so the experts can create lessons to share their knowledge with people who need it.
 
-![01_sharing_knowledge](https://d2mxuefqeaa7sj.cloudfront.net/s_5998CB8F4ACAE04353DF55D78578CAC9F495218C5B780DCB121ECA0E8390328B_1530529136099_file.png)
```

Comment:
@joelhooks @MaggieAppleton  could you please point me to the images that were used here? I couldn't find them anywhere.